### PR TITLE
More details on errors from node client

### PR DESCRIPTION
**Pull request by matteofigus**
Second part of #191.

If all is green, can you review and merge @pbazydlo ?

### CODE REVIEW

File: client/src/get-components-data.js [hunk 1]
```diff
@@ -48,10 +49,14 @@ module.exports = function(config){
     }
 
     makePostRequest(serverRendering.components, options, function(error, responses){
-      if(!!error || !responses || _.isEmpty(responses)){
+      if(!!error || !responses || _.isEmpty(responses)){ console.log(error);
```
Comment (andyroyle) [inline client/src/get-components-data.js]: is that `console.log` supposed to be there?

Comment (matteofigus) [inline client/src/get-components-data.js]: lol, no, thanks :dancer:

**Comment by andyroyle** [comment]
LGTM :+1:

**Comment by matteofigus** [comment]
Thanks @andyroyle

**Comment by pbazydlo** [comment]
@matteofigus didn't manage to do it in time, but it's hard to compete with @andyroyle :P

**Pull request merged**